--- novaclient/utils.py.old	2012-11-12 00:08:50.080615689 +0100
+++ novaclient/utils.py	2012-11-12 00:07:46.074178745 +0100
@@ -142,14 +142,14 @@
                 row.append(data)
         pt.add_row(row)
 
-    pt.printt(sortby=fields[0])
+    print(pt.get_string(sortby=fields[0]))
 
 
 def print_dict(d, property="Property"):
     pt = prettytable.PrettyTable([property, 'Value'], caching=False)
     pt.aligns = ['l', 'l']
     [pt.add_row(list(r)) for r in d.iteritems()]
-    pt.printt(sortby=property)
+    print(pt.get_string(sortby=property))
 
 
 def find_resource(manager, name_or_id):
